packages/wiz-ui-react/src/components/custom/timeline/components/timeline.tsx

```diff
@@ -0,0 +1,26 @@
+import { ComponentName } from "@wizleap-inc/wiz-ui-constants";
+import * as styles from "@wizleap-inc/wiz-ui-styles/customs/timeline.css";
+
+import { WizVStack } from "@/components/base";
+import { BreakpointVariant } from "@/hooks";
+
+import { TimelineContext } from "./use-timeline";
+
+import type { FC, ReactNode } from "react";
+
+interface TimelineProps {
+  bp: BreakpointVariant;
```

Comment:
[IMO] このコンポーネント単体で見た場合、この Props の意味は単純にサイズの大・中・小を指定するもので、ブレークポイントとは関心事を切り離して考えられるので、以下のような Props のにしたほうが意味が明確になって良いかもと思いました。

```suggestion
  size?: "sm" | "md" | "lg";
```